lib/redmine/scm/adapters/xitolite_adapter.rb

```diff
@@ -0,0 +1,572 @@
+require 'redmine/scm/adapters/abstract_adapter'
+
+# XitoliteAdapter inherits from GitAdapter but some classes which are define directly in GitAdapter are not inherited
+# (GitBranch, ScmCommandAborted and maybe others) so it raises NameError exception.
+# To fix this I had to reimplement (copy/past) the whole GitAdapter class in XitoliteAdapter...
+# I wanted to avoid it to avoid code duplication but it seems to be the only way...
+
+module Redmine
+  module Scm
+    module Adapters
+      class XitoliteAdapter < AbstractAdapter
+
+        # Git executable name
+        XITOLITE_BIN = Redmine::Configuration['scm_git_command'] || "git"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.